Makefile

```diff
@@ -36,7 +36,7 @@ vet:
 	go vet ./...
 
 # Build the docker image
-docker-build: test
+docker-build: 
```

Comment:
@mazocode Why do you remove the test??

Reply:
I've removed this on my test machine because it requires root to run:
`iptables_test.go:19: create chain IPv4 - out:iptables v1.8.4 (nf_tables): Could not fetch rule set generation id: Permission denied (you must be root)`
Reverted the change.